src/macros.rs

```diff
@@ -0,0 +1,288 @@
+/// Construct a `plist::Value` from a JSON literal.
+///
+/// ```
+/// # use plist::plist;
+/// #
+/// let value = plist!({
+///     "code": 200,
+///     "success": true,
+///     "payload": {
+///         "features": [
+///             "serde",
+///         ]
+///     }
+/// });
+/// ```
+///
+/// Variables or expressions can be interpolated into the PList literal. Any type
```

Comment:
Maybe "Variables or expressions can be interpolated into the literal"? There are no nice casings of "plist".